src/coreclr/jit/gentree.h

```diff
@@ -554,6 +554,7 @@ enum GenTreeFlags : unsigned int
 #ifdef FEATURE_HW_INTRINSICS
     GTF_HW_EM_OP                  = 0x10000000, // GT_HWINTRINSIC -- node is used as an operand to an embedded mask
```

Comment:
I don't think we need both `GTF_HW_EM_OP` and `GTF_HW_EMBEDDING_OP`, as `GTF_HW_EM_OP` can be implied when a node is contained and it's parent has `GTF_HW_EMBEDDING_OP`.

Didn't want to do that yet until the rest of the PR is agreed (or maybe push to a future PR)